src/Standards/Generic/Tests/CodeAnalysis/EmptyStatementUnitTest.inc

```diff
@@ -69,4 +69,18 @@ try {
     // TODO: Handle this exception later :-)
 }
 
-if (true) {} elseif (false) {}
\ No newline at end of file
+if (true) {} elseif (false) {}
+
+// phpcs:set Generic.CodeAnalysis.EmptyStatement allowComments true
+
+if ($foo) {
+    // Just a comment
+} elseif ($bar) {
+    /*
+     * Yet another comment
+     */
+} elseif ($baz) {
+    // phpcs:set Generic.CodeAnalysis.EmptyStatement somethingMadeUp true
```

Comment:
To prevent confusion between real test related PHPCS annotations and annotations which are part of the test, it's better to use something like `//phpcs:ignore Standard.Category.SniffName -- for reasons` here.